config/cmd/cmd.go

```diff
@@ -468,10 +467,79 @@ func (c *cmd) Options() Options {
 }
 
 func (c *cmd) Before(ctx *cli.Context) error {
-	// If flags are set then use them otherwise do nothing
-	var serverOpts []server.Option
+	// Setup client options
 	var clientOpts []client.Option
 
+	if r := ctx.Int("client_retries"); r >= 0 {
```

Comment:
`ctx.Int()` will return 0 in case `client_retries` is not set. Is that what we want? Not sure if there is a default client retries val

Reply:
The flag uses client.DefaultRetries as the default value, so it would be 1 by default.